### PR TITLE
fix: email domain and missing rcpt to

### PR DESCRIPTION
**Pull request by ssoroka**

### CODE REVIEW

File: internal/server/email/sendgrid.go [hunk 1]
```diff
@@ -65,6 +68,11 @@ func SendTemplate(name, address string, template EmailTemplate, data any) error
 		return ErrNotConfigured
 	}
 
+	if name == "" {
+		// until we have real user names
+		name = BuildNameFromEmail(address)
```
Comment (mxyng) [inline internal/server/email/sendgrid.go:73]: I don't think we need to over rotate on this. Instead, just set the name to the email until we have actual user names

Comment (ssoroka) [reply internal/server/email/sendgrid.go:73]: the experience is pretty nice vs email addresses. I created this function for another spot that uses the name in the mail body and looked a lot better than "joe.smith@example.com invited you to ...."

File: internal/server/handlers.go [hunk 2]
```diff
@@ -145,7 +145,7 @@ func (a *API) Signup(c *gin.Context, r *api.SignupRequest) (*api.SignupResponse,
 	a.t.Event("signup", identity.ID.String(), Properties{})
 
 	err = email.SendSignupEmail("", r.Name, email.SignupData{
-		Link: fmt.Sprintf("https://%s/login", r.Org.Subdomain),
+		Link: fmt.Sprintf("https://%s/login", suDetails.Org.Domain),
```
Comment (mxyng) [inline internal/server/handlers.go:148]: If I'm reading this correctly, `r.Org.Subdomain` only contains the subdomain part but the details contains a FQDN?

Comment (ssoroka) [reply internal/server/handlers.go:148]: yes

**Review by mxyng — APPROVED** [review]

File: internal/server/handlers.go (hunk 2 above)
Comment (mxyng) [reply internal/server/handlers.go:148]: Can be in a follow up PR but should add a test case for this

**Pull request merged**